apriltag.c

```diff
@@ -296,20 +297,20 @@ static void quick_decode_codeword(apriltag_family_t *tf, uint64_t rcode,
     struct quick_decode *qd = (struct quick_decode*) tf->impl;
 
     // quick_decode might be NULL if detector_add_family_bits() failed
-    for (int ridx = 0; qd!=NULL &&ridx < 4; ridx++) {
+    for (int ridx = 0; qd!=NULL && ridx < 4; ridx++) {
 
-            for (int bucket = rcode % qd->nentries;
-                qd->entries[bucket].rcode != UINT64_MAX;
-                bucket = (bucket + 1) % qd->nentries) {
+        for (int bucket = rcode % qd->nentries;
+            qd->entries[bucket].rcode != UINT64_MAX;
+            bucket = (bucket + 1) % qd->nentries) {
 
-                if (qd->entries[bucket].rcode == rcode) {
-                    *entry = qd->entries[bucket];
-                    entry->rotation = ridx;
-                    return;
-                }
+            if (qd->entries[bucket].rcode == rcode) {
+                *entry = qd->entries[bucket];
+                entry->rotation = ridx;
+                return;
             }
+        }
 
-            rcode = rotate90(rcode, tf->nbits);
+        rcode = rotate90(rcode, tf->nbits);
```

Comment:
This should all be part of your earlier commit "Tweak quick_decode check". Please "fixup" those commits.